pkg/imgpkg/cmd/copy_repo_src.go

```diff
@@ -77,17 +84,63 @@ func (c CopyRepoSrc) CopyToRepo(repo string) (*ctlimgset.ProcessedImages, error)
 		return nil, fmt.Errorf("Building import repository ref: %s", err)
 	}
 
+	c.logger.Debugf("copy the fetched images\n")
 	processedImages, ids, err := c.imageSet.Relocate(unprocessedImageRefs, importRepo, c.registry)
 	if err != nil {
 		return nil, err
 	}
 
+	if c.originBundle != nil {
```

Comment:
per our discussion, consolidating this functionality on the Bundle would nicely keep consumers of bundles oblivious to image locations.

Reply:
fixed